helpers/collection/collection.go

```diff
@@ -38,14 +38,14 @@ type (
 	// Options to Format list-response
 	Options struct {
 		Query   *orm.Query
-		Fetcher ModelsFetcher
-		Filter  Filter
+		Fetcher Fetcher
+		Former  Former
 	}
 )
 
-// FormatQuery collection as response-answer (Collection)
-func FormatQuery(ctx web.Context, opts Options) error {
-	if opts.Query == nil || opts.Filter == nil || opts.Fetcher == nil {
+// ResponseQuery collection as response-answer (Collection)
+func ResponseQuery(ctx web.Context, opts Options) error {
```

Comment:
I'm vote for `Response`, without `Query`
since it makes no sense in `Query` part of naming